src/hooks/useConstitutionLinter.ts

```
@@ -0,0 +1,116 @@
+            // 2. Insert record into constitution_documents
+            const { data: docRecord, error: insertError } = await supabase
+                .from('constitution_documents')
+                .insert({
+                    club_id: clubId,
+                    file_url: publicUrl,
+                    status: 'pending_review'
+                })
+                .select()
+                .single();
+
+            if (insertError || !docRecord) throw insertError;
```

ESLint

[error] Delete `······`

(prettier/prettier)